CMakeLists.txt

```diff
@@ -140,6 +140,7 @@ configure_file(
 
 ################################################################
 
+link_libraries(stdc++fs)
```

Comment:
I tried this previously but had lots of trouble on mac.  Hopefully this will work ok a few years later.

Reply:
C++ makes me sad